examples/repoless/default-ns.yaml

```diff
@@ -15,4 +15,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: default
+  name: backend-default
```

Comment:
Just curious why do we want to change the namespace? If so, we need to update the file name as it is not the default namespace anymore,

Reply:
ACK